e2e/cypress/support/commands.ts

```diff
@@ -202,6 +202,49 @@ Cypress.Commands.add('cleanupProjectsByIDPrefix', (idToken: string, idPrefix: st
   });
 });
 
+Cypress.Commands.add('cleanupRolesByIDPrefix', (idToken: string, idPrefix: string) => {
```

Comment:
TODO: Gonna make a general version of all these cleanup commands in the AM, so ignore while reviewing tonight.